legend-engine-language-pure-dsl-service/src/test/java/org/finos/legend/engine/language/pure/grammar/test/TestServiceGrammarParser.java

```diff
@@ -1164,23 +1164,6 @@ public void testServiceTelemetry() {
                 "    metricGroups: ['group1', 'group2'];\n" +
                 "    metricGroups: ['group1', 'group2'];\n" +
                 "  }\n" +
-                "}\n","PARSER error at [17:5-16]: Unexpected token");
-        test("###Service\n" +
-                "Service meta::pure::myServiceSingle\n" +
-                "{\n" +
-                "  pattern: 'url/myUrl/';\n" +
-                "  owners: ['ownerName', 'ownerName2'];\n" +
-                "  documentation: 'this is just for context';\n" +
-                "  autoActivateUpdates: true;\n" +
-                "  execution: Single\n" +
-                "  {\n" +
-                "    query: |demo::_NPerson.all()->graphFetch(#{demo::_NPerson{Age,Name}}#)->serialize(#{demo::_NPerson{Age,Name}}#);\n" +
-                "    mapping: meta::myMapping;\n" +
-                "    runtime: meta::myRuntime;\n" +
-                "  }\n" +
-                "  telemetry:\n" +
-                "  {\n" +
-                "  }\n" +
-                "}\n");
+                "}\n","PARSER error at [14:3-18:3]: Field 'metricGroupsTelemetry' should be specified only once");
```

Comment:
there is no such field like metricGroupsTelemetry. It would be confusing for user. It should just say metricGroups.